pages/getting-started/installation.md

```diff
@@ -17,24 +17,18 @@ https://docs.conda.io/projects/conda/en/latest/user-guide/install/
 3. Activate the conda environment:<br/>
 `conda activate fisher`
 4. Add the Bioconda, Conda-Forge, & PhyloFisher Anaconda Cloud channels to your channels:<br/>
-`conda config --append channels phylofisher`<br/>
 `conda config --append channels bioconda`<br/>
 `conda config --append channels conda-forge`<br/>
-5. Install PhyloFisher:<br/>
+1. Install PhyloFisher:<br/>
 `mamba install phylofisher`
 
 Notes:
 - After you finish using PhyloFisher, use `conda deactivate` to deactivate the `fisher` conda virtual enviornment.
 <br/>
 
 
-## Installation of `forest.py` & ParaSorter locally (including Windows)
+## Installation of `forest_local.py` & ParaSorter locally (including Windows)
 
-Installation of these two tools on a users local machine is required for downstream steps in the PhyloFisher
-workflow. Although the entire PhyloFisher software package does not run on Windows, this minimal
-download will. This download will provide users (including Windows users) the graphical tree parsing
-aspects of the package on their local machines. This minimal download of `forest.py` and ParaSorter can be
-retrieved from the ”ParaSorter” section of PhyloFisher GitHub Repository.
+Installation of `forest_local.py` (a standalone version of `forest.py`) and ParaSorter on a users local machine is required for downstream steps in the PhyloFisher workflow. Although the entire PhyloFisher software package does not run on Windows, this minimal download will. This download will provide users (including Windows users) the graphical tree parsing aspects of the package on their local machines. This minimal download of `forest.py` and ParaSorter can be retrieved from the [”ParaSorter” section](https://github.com/TheBrownLab/PhyloFisher/tree/master/parasorter) of PhyloFisher GitHub Repository.
```

Comment:
Should forest.py on the next to last line be forest_local.py?

Reply:
Yes